_includes/ios/objects.md

```diff
@@ -461,78 +471,75 @@ You can also model a many-to-many relation using the `PFRelation` object.  This
 PFUser *user = [PFUser currentUser];
 PFRelation *relation = [user relationForKey:@"likes"];
 [relation addObject:post];
-[user saveInBackgroundWithBlock:^(BOOL succeeded, NSError *error) {
-  if (succeeded) {
-    // The post has been added to the user's likes relation.
-  } else {
-    // There was a problem, check error.description
-  }
+[user saveInBackgroundWithBlock:^(BOOL succeeded, NSError * _Nullable error) {
+    if (succeeded) {
+        // The post has been added to the user's likes relation.
+    } else {
+        // There was a problem, check error.description
+    }
 }];
 ```
 ```swift
-let user = PFUser.current()
-let relation = user?.relation(forKey: "likes")
-relation?.add(post)
-user?.saveInBackground(block: { (success: Bool, error: Error?) in
-    if (success) {
+guard let user = PFUser.current() else { return }
+let relation = user.relation(forKey: "likes")
+relation.add(post)
+user.saveInBackground { (succeeded, error) in
+    if (succeeded) {
         // The post has been added to the user's likes relation.
     } else {
         // There was a problem, check error.description
     }
-})
-
+}
 ```
 </div>
 
-You can remove a post from the `PFRelation` with something like:
-
-<div class="language-toggle" markdown="1">
-```objective_c
-[relation removeObject:post];
-```
-```swift
-relation?.remove(post)
-```
-</div>
+You can remove a post from the `PFRelation` similarly using the `removeObject:` function followed by saving the parent object.
```

Comment:
Are you planning to favour the swift methods signatures as we discussed? or leaving this to another PR?

Reply:
Unfortunately, by the time I thought about it and got your input these changes had already been committed.

Reply:
Ok, maybe we’ll leave this for a future pr?

Reply:
For sure. My plan is to kind of get through the rest of this and then return to update the docs with the swift functions as well as switch swift to the default example.

Reply:
Sounds good 👍